owrx/controllers/settings/general.py

```diff
@@ -117,6 +118,11 @@ def getSections(self):
                     infotext="Specifies the upper and lower dynamic headroom that should be added when automatically "
                     + "adjusting waterfall colors",
                 ),
+                CheckboxInput(
+                    "waterfall_auto_level_default_mode",
+                    'Automatically adjust waterfall level by default',
+                    infotext="Enable this to automatically enable auto adjusting waterfall levels on page load."
+                ),
```

Comment:
I think it would be a good idea to have this input repeated on the device and profile pages (same as with the waterfall levels) so that the default can be overridden per profile (for example if you have a mixed HF and VHF / UHF receiver, the HF profiles will probably work better with it on, whereas on VHF / UHF, it may be better to have it off).